src/main/java/io/vertx/gsoc2018/qotd/QuoteOfTheDayVerticle.java

```diff
@@ -32,19 +51,28 @@ public void start(Future<Void> startFuture) throws Exception {
 
   private Future<Void> runScript(String script) {
     Future<Void> future = Future.future();
-    jdbcClient.getConnection(getConn -> {
-      if (getConn.succeeded()) {
-        SQLConnection connection = getConn.result();
-        connection.execute("RUNSCRIPT FROM '" + script + "'", exec -> {
-          connection.close();
-          if (exec.succeeded()) {
-            future.succeeded();
+
+    vertx.fileSystem().readFile(script, fileReadResult -> {
```

Comment:
no doubt this works but why did you remove the `RUNSCRIPT` version? it's much more concise and really enough for our use case. just curious

Reply:
oh, sorry, I simply not familiar with `RUNSCRIPT` command and I thought that the command is like `TODO` and should be replaced